### ISSUE TITLE
Console Error: Moment.js Value Not Provided In RFC2822/ISO Format

### ISSUE THREAD

**Issue by tparikka** [ThatConference/that-website#313]
## Expected Behavior
Moment.js expects values to be provided in an RFC2822 or ISO format.

## Current Behavior
The value is provided in an invalid format resulting in a fallback to js Date():

```
Deprecation warning: value provided is not in a recognized RFC2822 or ISO format. moment construction falls back to js Date(), which is not reliable across all browsers and versions. Non RFC2822/ISO date formats are discouraged and will be removed in an upcoming major release. Please refer to http://momentjs.com/guides/#/warnings/js-date/ for more info.
Arguments: 
[0] _isAMomentObject: true, _isUTC: false, _useUTC: false, _l: undefined, _i: 11/01/19, _f: undefined, _strict: undefined, _locale: [object Object]
D/<@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:547274
Et/</<@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:567179
Et/<@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:567230
Et@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:567524
Ht/<@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:567914
Ht@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:567976
It@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:568008
a@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:544109
WhJq/t.a</r<@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:346797
pa/</<.criteria<@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:218359
Xt@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:200715
pa/<@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:218334
la/<@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:217323
ii/<@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:223634
la@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:217296
pa@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:218296
e/Ks<@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:236782
Vt@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:199950
$i/<@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:232144
Fa/<@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:221446
tn@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:200897
Fa@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:221412
e/_r.prototype.value@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:266202
WhJq/t.a<@https://www.thatconference.com/_next/static/that-website@1.0.0-ed79e2/pages/wi.js:1:346808
ha@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:338281
Fu@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:383717
jl@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:369239
Pl@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:369162
xl@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:366482
ml@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:363226
oc@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:391300
sc/<@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:392537
_l@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:366833
sc@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:392522
hydrate@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:394365
e/<@https://www.thatconference.com/_next/static/runtime/main-0f3660c2198e434f6af5.js:1:9569
c@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:192903
l/i._invoke</<@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:192692
k/</e[t]@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:193326
a@https://www.thatconference.com/_next/static/runtime/main-0f3660c2198e434f6af5.js:1:137
i@https://www.thatconference.com/_next/static/runtime/main-0f3660c2198e434f6af5.js:1:337
+oT+/e.exports/</<@https://www.thatconference.com/_next/static/runtime/main-0f3660c2198e434f6af5.js:1:396
t@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:138428
+oT+/e.exports/<@https://www.thatconference.com/_next/static/runtime/main-0f3660c2198e434f6af5.js:1:282
de@https://www.thatconference.com/_next/static/runtime/main-0f3660c2198e434f6af5.js:1:10007
fe@https://www.thatconference.com/_next/static/runtime/main-0f3660c2198e434f6af5.js:1:8791
e/<@https://www.thatconference.com/_next/static/runtime/main-0f3660c2198e434f6af5.js:1:6151
c@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:192903
l/i._invoke</<@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:192692
k/</e[t]@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:193326
a@https://www.thatconference.com/_next/static/runtime/main-0f3660c2198e434f6af5.js:1:137
i@https://www.thatconference.com/_next/static/runtime/main-0f3660c2198e434f6af5.js:1:337
+oT+/e.exports/</<@https://www.thatconference.com/_next/static/runtime/main-0f3660c2198e434f6af5.js:1:396
t@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:138428
+oT+/e.exports/<@https://www.thatconference.com/_next/static/runtime/main-0f3660c2198e434f6af5.js:1:282
re@https://www.thatconference.com/_next/static/runtime/main-0f3660c2198e434f6af5.js:1:6322
te@https://www.thatconference.com/_next/static/runtime/main-0f3660c2198e434f6af5.js:1:5893
e/<@https://www.thatconference.com/_next/static/runtime/main-0f3660c2198e434f6af5.js:1:5686
c@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:192903
l/i._invoke</<@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:192692
k/</e[t]@https://www.thatconference.com/_next/static/chunks/commons.6fc88875d1764300b13a.js:1:193326
a@https://www.thatconference.com/_next/static/runtime/main-0f3660c2198e434f6af5.js:1:137
i@https://www.thatconference.com/_next/static/runtime/main-0f3660c2198e434f6af5.js:1:337
```

## Possible Solution
Provide a value to the Moment.js constructor that is RC2822 or ISO compliant.

## Steps to Reproduce
1. Navigate to https://www.thatconference.com/wi in Firefox, Chrome, or Edge in Win10 Pro.

**Comment by brettski** [comment]
Fixed with PR #378

**Comment by brettski** [comment]
Thank you @tparikka for filing this bug!
